code/modules/client/preferences_toggle_procs.dm

```diff
@@ -152,7 +152,7 @@
 /client/verb/toggle_admin_midis()
 	set name = "Toggle Admin MIDIs"
 	set category = "Preferences"
-	set desc = "Toggles the music in the lobby."
+	set desc = "Toggles the ability to hear the music in the lobby."
```

Comment:
Name and desc should probably be changed wholesale. Admins can do a lot more than play midi files nowadays, and it sure as hell ain't just lobby music.

Reply:
I somehow missed that.